backend/open-graph-api/src/main/java/com/woowacourse/taggle/crawler/OpenGraphCrawler.java

```diff
@@ -19,28 +19,49 @@
     private final static String META_IMAGE = "meta[property=og:image]";
     private final static String URL_REGEX = "^(https?)://?[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]";
     private final static Pattern pattern = Pattern.compile(URL_REGEX);
+    private final static String CONTENT = "content";
+
+    private final String defaultImage;
+
+    public OpenGraphCrawler(String defaultImage) {
+        this.defaultImage = defaultImage;
+    }
+
+    public OpenGraph extractPreview(String url) {
+        validateURL(url);
+        try {
+            Document document = jsoupConfig(url);
+            String title = getTitle(document);
+            String description = document.select(META_DESCRIPTION).attr(CONTENT);
```

Comment:
가독성과 다른 메서드와의 일관성을 위해 메서드로 분리해보는건 어떨까요?
```suggestion
            String description = getDescription(document);
```